spec/operations/story_operations/create_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+describe StoryOperations::Create do
+  let(:story_params) do
+    { title: 'Test Story', requested_by: user, state: 'unstarted', accepted_at: nil }
+  end
+
+  let!(:membership) { create(:membership) }
+  let(:user)        { membership.user }
+  let(:project)     { membership.project }
+  let(:story)       { project.stories.build(story_params) }
+
+  subject { -> { StoryOperations::Create.new.call(story: story, current_user: user) } }
+
+  context 'with valid params' do
+    it { expect { subject.call }.to change { Story.count } }
+    it { expect { subject.call }.to change { Changeset.count } }
+    it { expect(subject.call.value!).to be_eql Story.last }
+
+    Story::ESTIMABLE_TYPES.each do |story_type|
+      context "a #{story_type} story" do
+        it 'keeps the story state unscheduled' do
+          story.attributes = { state: 'unscheduled', story_type: story_type, estimate: 1 }
+          subject.call
+          expect(story.state).to eq('unscheduled')
+        end
+      end
+
+      context "a started #{story_type} story" do
+        it 'sets the story state as started' do
+          story.attributes = { state: 'started', story_type: story_type, estimate: 1 }
+          subject.call
+          expect(story.state).to eq('started')
+        end
+      end
+    end
+
+    context 'a non estimable story' do
+      it 'keeps the story state as unscheduled' do
+        story.attributes = { state: 'unscheduled', story_type: 'release' }
+        subject.call
+        expect(story.state).to eq('unscheduled')
+      end
+    end
+  end
+
+  context 'with invalid params' do
+    before { story.title = '' }
+
+    it { is_expected.to_not change { Story.count } }
+    it { expect(subject.call.success?).to be_falsy }
+    it { expect(Notifications).to_not receive(:story_mention) }
+  end
+
+  context '::UserNotification' do
+    let(:mailer) { double('mailer') }
```

Comment:
which mailer this is being double of? you can specify like this for instance (I don't know which mailer is being used, so this might fail) `double(NotificationMailer, deliver_later: true)` and will work, also we can tell which mailer is being used for this case here.